app/assets/javascripts/discourse/tests/acceptance/sidebar-anonymous-community-section-test.js

```diff
@@ -83,3 +83,34 @@ acceptance("Sidebar - Anonymous user - Community Section", function (needs) {
       );
   });
 });
+
+acceptance(
+  "Sidebar - Anonymous user - Community Section with hidden links",
+  function (needs) {
+    needs.settings({ navigation_menu: "sidebar" });
+
+    needs.site({
+      anonymous_sidebar_sections: [
+        {
+          id: 1,
+          title: "The A Team",
+          section_type: "community",
```

Comment:
We need this `section_type` to ensure it's considered as the `community` section.